templates/commands/render.go

```diff
@@ -540,13 +540,13 @@ func destOK(fs fs.StatFS, dest string) error {
 	fi, err := fs.Stat(dest)
 	if err != nil {
 		if errors.Is(err, os.ErrNotExist) {
-			return fmt.Errorf("the destination directory doesn't exist: %q", dest)
+			return nil
```

Comment:
Should we document that its ok to not exists because the render process will generate it?

Reply:
Done.